examples/credential_metadata_mso_mdoc.json

```diff
@@ -33,36 +31,41 @@
           "text_color": "#FFFFFF"
         }
       ],
-      "claims": {
-        "org.iso.18013.5.1": {
-          "given_name": {
-            "display": [
-              {
-                "name": "Given Name",
-                "locale": "en-US"
-              },
-              {
-                "name": "名前",
-                "locale": "ja-JP"
-              }
-            ]
-          },
-          "family_name": {
-            "display": [
-              {
-                "name": "Surname",
-                "locale": "en-US"
-              }
-            ]
-          },
-          "birth_date": {
-            "mandatory": true
-          }
+      "claims": [
+        {
+          "namespace": "org.iso.18013.5.1",
+          "claim_name": "given_name",
```

Comment:
I think using `path` for mdoc as well and use namespace+cliam_name as two array elements seems cleaner to me

Reply:
Discussed on the WG call today: tendency to do the change and do it now in VP and VCI.